.github/workflows/auto-release.yml

```diff
@@ -10,6 +10,7 @@ permissions:
 
 jobs:
   release:
+    if: ${{ github.ref == 'refs/heads/master' }} && false
```

Comment:
I think you need to drop `false` here. And, you can then just put the conditional without brackets.